### PR TITLE
intial update to include textTag api for java

### PR DESCRIPTION
**Pull request by anneCarlson**
General changes to make the text tag api callable in java. Still need to update the readme.

### CODE REVIEW

**Comment by anneCarlson** [comment]
Need to update codeship to have some environment variables. This now does batch requests and private cloud requests as well.

File: README.md [hunk 1]
```diff
@@ -76,16 +77,94 @@ or Gradle:
...
+
+```
+$INDICO_API_KEY
+$INDICO_CLOUD
+```
+
+Variables set with a `Indico.setPrivateCloud(String cloud)` or `Indico.setApiKey(String apiKey)` override any environment variables or configuration found in the indicorc file for any subsequent api calls, like so:
+
+```java
+Indico.setApiKey(null);
```
Comment (madisonmay) [inline README.md]: Why are we setting these to null in the examples?  And then why does our example show the call to sentiment working afterwards?

File: src/main/java/io/indico/api/Config.java [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+import java.io.File;
+
+import org.ini4j.Ini;
+
+import io.indico.api.ConfigLogic;
+import io.indico.api.exception.IndicoException;
+
+
+public class Config{
     public final static String API_URL = "http://apiv1.indico.io";
```
Comment (madisonmay) [inline src/main/java/io/indico/api/Config.java]: We should change this url to HTTPS and update to apiv2.indico.io

File: README.md (hunk 1 above)
Comment (anneCarlson) [inline README.md]: Good catch! I'll pull that and an example api key change shortly.

File: src/main/java/io/indico/api/Config.java (hunk 2 above)
Comment (anneCarlson) [inline src/main/java/io/indico/api/Config.java]: Yay!

Comment (anneCarlson) [inline src/main/java/io/indico/api/Config.java]: Worth noting. We currently have `http://indico-test.indico.domains` (which we should also to update to https, correct?) pointing at apiv1. We should have it redirect to v2 to make tests pass.

Comment (madisonmay) [inline src/main/java/io/indico/api/Config.java]: Good call.

**Comment by anneCarlson** [comment]
Updated!

**Comment by madisonmay** [comment]
Looks good to me!  Rebase and then I think we're good to merge once the apiv2.indico.io setup is deployed.

**Comment by madisonmay** [comment]
Do we need to update our codeship build to use java 7?

**Comment by madisonmay** [comment]
A few portions are still broken due to testing private cloud over SSL.  Not quite sure how we want to handle that one yet... I think for testings sake we can change the protocol to HTTP rather than HTTPS to avoid raising SSL errors when our mock private cloud redirects to our public API.

**Pull request merged**